### ISSUE TITLE
Extension does not use display language of vscode

### ISSUE THREAD

**Issue by Balkoth** [microsoft/vscode-mssql#17540]
- MSSQL Extension Version: 1.17.1
- VSCode Version: 1.75.0
- OS Version: Windows 10 21H2 19044.2486

Steps to Reproduce:

1. Use VSCode on a system with another language than en-us
2. Do not install any language packs and let VSCode use en-us as display language
3. Watch how this extension thinks it knows better and use the language provided by the os for its localization

I use en-us for a reason in my IDE, so imho it's a bad idea to ignore my decision.

**Comment by Balkoth** [comment]
Problem persists to this day.

**Comment by bennoremec** [comment]
Jup, especialy with the suboptimal translations, i prefer to show it up in english.

**Comment by chris-waitman** [comment]
Hi,
The problem still exists. When will this finally be fixed? The same nonsense with the mixed languages also happens in the Oracle SQL Developer Extension. This is a terrible experience!

**Comment by thomas-broethaler** [comment]
Yes I would also really appreciate it using the language set via "Configure Display Language" (which i currently set to english, but I see german objects as mentioned by OP). Especially since we will have to move to this extension eventually when Azure Data Studio will be deprecated.

**Comment by kburtram** [comment]
It looks like this might be related to our out-of-proc component detecting the locale independently, rather than inheriting it from the VS Code process. The incorrectly localized strings seem to be originating from there. We’ll take a closer look and fix.
